packages/airnode-deployer/src/cli/index.ts

```diff
@@ -193,6 +194,28 @@ yargs(hideBin(process.argv))
       await listAirnodes(args.cloudProviders);
     }
   )
+  .command(
+    'info <deployment-id>',
+    'Displays info about deployed Airnode',
+    (yargs) => {
+      yargs.positional('deployment-id', {
+        description: `ID of the deployment (from 'list' command)`,
+        type: 'string',
+        demandOption: true,
+      });
+    },
+    async (args) => {
+      logger.debugMode(args.debug as boolean);
+      logger.debug(`Running command ${args._[0]} with arguments ${longArguments(args)}`);
+
+      // Looks like due to the bug in yargs (https://github.com/yargs/yargs/issues/1649) we need to specify the type explicitely
+      const goDeploymentInfo = await go(() => deploymentInfo(args.deploymentId as string));
+      if (!goDeploymentInfo.success) {
+        // eslint-disable-next-line functional/immutable-data
+        process.exitCode = 1;
```

Comment:
Isn't this too harsh? Will there be a message logged out?

Reply:
Yes, there will be https://github.com/api3dao/airnode/blob/deployment-info/packages/airnode-deployer/src/infrastructure/index.ts#L694
I think it should end up with the non-zero exit code when it can't find the deployment. It's the same thing as when you call `docker inspect <ID>` with an invalid/non-existent ID.